docs/pip.md

```diff
@@ -716,97 +718,27 @@ export PIP_FIND_LINKS=/tmp/hermeto-output/deps/pip
 export PIP_NO_INDEX=true
 ```
 
-#### Inject project files
-
-In order to be able to install pip dependencies in a hermetic environment, we
-need to perform the injection to change the remote dependencies to instead point
-to the local file system.
-
-```shell
-$ hermeto inject-files ./hermeto-output --for-output-dir /tmp/hermeto-output
-2023-01-26 16:41:09,990 INFO Overwriting /tmp/test/atomic-reactor/requirements.txt
-```
-
-We can look at the `git diff` to see what the package remapping looks like. As
-an example,
-
-```diff
-diff --git a/requirements.txt b/requirements.txt
--osbs-client @ git+https://github.com/containerbuildsystem/osbs-client@8d7d7fadff38c8367796e6ac0b3516b65483db24
--    # via -r requirements.in
-+osbs-client @ file:///tmp/hermeto-output/deps/pip/github.com/containerbuildsystem/osbs-client/osbs-client-external-gitcommit-8d7d7fadff38c8367796e6ac0b3516b65483db24.tar.gz
-```
-
-*⚠ This is only needed for [external dependencies][].
-If all dependencies come from PyPi, Hermeto will not replace anything.*
-
-#### Build the base image
-
-For this example, we will split the build into two parts - a base image and the
-final application image. Since there is no way to install RPMs in a hermetic
-environment, we will create the base image with its required "devel" libraries
-from RPMs in one image and then use that image for our hermetic python build.
-
-If your project doesn't need to compile as many C packages as atomic-reactor,
-you may be able to find a base image that already contains everything you need.
-
-Dockerfile.baseimage
-
-```dockerfile
-FROM quay.io/centos/centos:stream8
-
-# python3.8 runtime, C build dependencies
-RUN dnf -y install \
-        python38 \
-        python38-pip \
-        python38-devel \
-        gcc \
-        make \
-        libffi-devel \
-        krb5-devel \
-        cairo-devel \
-        cairo-gobject-devel \
-        gobject-introspection-devel \
-        openssl-devel && \
-    dnf clean all
-```
-
-This container build might be what we are familiar with already as we are not
-using Hermeto or enforcing network isolation.
-
-```shell
-podman build . -f Dockerfile.baseimage --tag atomic-reactor-base-image:latest
-```
-
 #### Build the application image
 
-We will base the final application image on our custom base image. The base
-image build installed all the RPMs we will need, so the final phase can use
-network isolation again 🎉. In order to support the network isolated build, we
-need to remember to `source` the environment file in the step that executes `pip
-install`. Because `osbs-client` comes from GitHub, the source code in
-`/src/atomic-reactor` has also been changed so that the dependencies are
-pointing to the cached versions.
+In order to support the network isolated build, we need to remember to `source`
```

Comment:
Do we want to include the inject-files subcommand in this example (for completeness) even if it isn't strictly needed in this specific case (because there are no non-registry dependencies)?

The build.sh script in the pip-basic example uses the inject files command

Reply:
6 of one, 1/2 dozen the other. I think I'd rather remove the command from the shell script: it's confusing to have a noop without calling it out, but then that itself is a little silly.

I fully intend to add a more complex example to the doc-examples repo at some point, which would (hopefully) demonstrate all of these features. For that, based on what I've seen so far, I will almost certainly need @a-ovchinnikov's help with building Rust from pip.